media/public/include/IWebAudioPlayer.h

```diff
@@ -29,12 +29,13 @@
  * current audio output.
  */
 
-#include "IWebAudioPlayerClient.h"
```

Comment:
If you want to change include order only, I'd keep the old one to avoid public interface change

Reply:
Does this actually change the interface? And the update conforms to the google coding standards

Reply:
ok, so you can change that